README.md

```diff
@@ -3,7 +3,6 @@
 | React Output Target   | [`@stencil/react-output-target`](https://www.npmjs.com/package/@stencil/react-output-target)     | [![version](https://img.shields.io/npm/v/@stencil/react-output-target/latest.svg)](https://www.npmjs.com/package/@stencil/react-output-target)     | [README](./packages/react-output-target/README.md)   |
 | Angular Output Target | [`@stencil/angular-output-target`](https://www.npmjs.com/package/@stencil/angular-output-target) | [![version](https://img.shields.io/npm/v/@stencil/angular-output-target/latest.svg)](https://www.npmjs.com/package/@stencil/angular-output-target) | [README](./packages/angular-output-target/README.md) |
 | Vue Output Target     | [`@stencil/vue-output-target`](https://www.npmjs.com/package/@stencil/vue-output-target)         | [![version](https://img.shields.io/npm/v/@stencil/vue-output-target/latest.svg)](https://www.npmjs.com/package/@stencil/vue-output-target)         | [README](./packages/vue-output-target/README.md)     |
-| Svelte Output Target  | [`@stencil/svelte-output-target`](https://www.npmjs.com/package/@stencil/svelte-output-target)   | [![version](https://img.shields.io/npm/v/@stencil/svelte-output-target/latest.svg)](https://www.npmjs.com/package/@stencil/angular-output-target)  | [README](./packages/svelte-output-target/README.md)  |
```

Comment:
If someone from the community takes ownership of this repo, I'll probably add something to the effect of 

> Looking for the Svelte output target? It's now maintained by the community. Click here to check it out!